src/react-aria-modal.js

```diff
@@ -166,9 +167,9 @@ let Modal = class Modal extends React.Component {
       dialogProps.tabIndex = '-1';
     }
 
-    const childrenArray = [React.DOM.div(dialogProps, props.children)];
+    const childrenArray = [DOM.div(dialogProps, props.children)];
```

Comment:
Not sure, just followed the instructions on the FB/React page. React.createElement is probably fine as well. It's kind of the same syntax right? Why wasn't that used instead of React.DOM.div?